src/plugins/messenger/MessengerPreview/components/MessengerButton/MessengerButton.tsx

```diff
@@ -14,33 +14,43 @@ interface IMessengerButtonProps {
 
 export const getMessengerButton = ({ React, styled }: MessagePluginFactoryProps) => {
 
-    const Button = styled.button(({ theme }) => ({
+    const ComponentStyles = (({ theme }) => ({
```

Comment:
ComponentStyles is not an element, should start with lowercase letter

Reply:
Fixed